ZecHubInterview.md

```diff
@@ -0,0 +1,126 @@
+* Who might be a good person to interview for this piece?
+
+
+---
+--- ARTICLE GOES BELOW THIS POINT ---
+
+---
+
+# Article Title
+
+| Content | Description |
+|---|---|
+| Title               | FIXME |
+| Author              | FIXME |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| Publish date        | YYYY-MM-DD FIXME |
+| Ai checks (https://www.zerogpt.com) | Score must be lower than 25% or article is rejected for publication |
+
+| Social Media | Links |
+|---|---|
+| Bison Relay post    | Title |
+| Reddit link         | FIXME |
+| Tweeter link        | FIXME |
+| TwitterSpace link   | FIXME |
+| Matrix post         | Promote your main tweet in "DCR Social Media" channel |
+
+## Thesis statement
+
+What the article is about, and what is it aiming to prove?
+
+
+## Pre-discussion, research and questions
+
+* What questions are you aiming to find the answers to?
+* Show links to research, which statements or quotes are you hoping to use?
+* Who might be a good person to interview for this piece?
+
+
+---
+--- ARTICLE GOES BELOW THIS POINT ---
+
+---
+
+# Interview with ZecHub
+
+First I want to thank ZecHub for participating! How are you? Please introduce yourself! Talk about your origins on the crypto market and, of course, on the Zcash community.
+
+Hey! My name is squirrel. I am currently a core contributor on the ZecHub team. My introduction to crypto came in early 2017 when a friend of mine demonstrated a transaction and then showed it in a block explorer. Since then I did everything possible to learn about blockchains such as certifications in Hyperledger, Blockchain Security and even made projects for myself such as creating a fork of Litecoin. 
+In 2022 I found a Request for Proposal (RFP) in the Zcash Community Forum. It was for a community notetaker role for the R&D meetings. Later on I joined ZecHub to write documentation and build the project to where it is today.  
+
+## Can you briefly describe what ZecHub is and its mission within the crypto community?
+
+ZecHub is an open source education hub for Zcash. We host a wiki site that has information for new zcash users looking to find out about wallet features, non-custodial exchanges, zero knowledge explainers and more. Global Web3 contributors are able to earn ZEC by writing documentation, creating graphic design elements such as posters or video animations, develop code for our network stats dashboard etc. Our DAO votes on project ideas that come in. 
+
+## What inspired the creation of ZecHub, and how does it aim to impact the broader Zcash ecosystem?
+
+ZecHub was founded by @Januszg_ on X. It was created as a fork of EthHub. The initial goal was to build a place new users could get started using Zcash. In parallel a podcast was created that seeks to build bridges between different projects in the privacy ecosystem such as Railgun & Decred.  
+Give the potential onset of a crypto bull run, how do you see the current state of the market?
```

Comment:
I think this needs to be reformatted as I believe it's a new question?

"Give the potential onset of a crypto bull run, how do you see the current state of the market?" The grammar for this question isn't quite correct maybe "With the potential onset of a crypto bull run ..."